content/blog/2020/06/camel-k-release-1.0.0/index.md

```diff
@@ -0,0 +1,250 @@
+
+We've added much more features for having a better integration with the Knative ecosystem and we've also fixed some compatibility and performance issues that were present in previous versions. The user experience is now much smoother.
+
+If you are a Knative YAML developer (!), instead of using Camel K directly, you also have the option to use **[Knative Camel Sources](https://knative.dev/docs/eventing/samples/apache-camel-source/)** which are part of the Knative release. They are wrappers for Camel K integrations that are compatible with all the tools used by Knative developers (such as the `kn` CLI or the OpenShift serverless console).
+Sources in Knative can only push data into the various Knative endpoints, but not the other way around (i.e. they cannot be used to publish data from Knative to the outside).
+In Camel K you don't have this distinction: the Route is the fundamental building block of a Camel integration and you can do whatever you want with it. 
+
+## Fast startup and low memory
+
+We cannot say we're serverless without mentioning the work that we've been doing in improving the performance of Camel K integrations.
+
+Starting from **Camel 3.3.0** which is the default version used by Camel K 1.0.0, you can benefit from all improvements that have been made directly in the core to make it much more lightweight. Claus Ibsen has written a series of blog posts ([part 1](http://www.davsclaus.com/2020/01/apache-camel-31-more-camel-core.html?m=1), [part 2](http://www.davsclaus.com/2020/01/apache-camel-31-more-camel-core_30.html?m=1), [part 3](http://www.davsclaus.com/2020/02/apache-camel-31-more-camel-core.html?m=1), [part 4](http://www.davsclaus.com/2020/03/apache-camel-32-reflection-free.html?m=1)) to highlight what has been changed in the 3.x Camel timeline to reduce memory footprint and speedup the startup time, which is foundamental when running integrations in a serverless environment.
+
+But improvements are not only limited to the Camel core: we're doing much more. Several months ago we've started a new subproject of Apache Camel named ["Camel Quarkus"](https://github.com/apache/camel-quarkus) with the goal of seamlessly running integrations on top of the Quarkus framework. As you probably know, Quarkus is able to reduce the memory footprint of Java applications and improve the startup time, because it moves much startup logic to the build phase. And Quarkus applications can be also compiled to a native binary, allowing a dramatic improvements in performance.
+
+In Camel K 1.0.0 we support **Camel Quarkus in JVM mode**. The goal is to have also the in-cluster automatic native compilation soon (for some DSL languages, such as YAML), in one of next releases!
+
+To use Quarkus as underlying runtime, you just need to enable the **[Quarkus trait](https://camel.apache.org/camel-k/latest/traits/quarkus.html)** when running an integration:
+
+```
+kamel run myintegration.groovy -t quarkus.enabled=true
+```
```

Comment:
Quarkus is expected to be the default underlying runtime in the next release, and support for Standalone mode (via camel-main) will be deprecated and removed.